plugins/console/init.lua

```diff
@@ -225,6 +234,16 @@ return ln.linenoise('$PROMPT')
 	emu.register_stop(function() consolebuf = nil end)
 
 	emu.register_periodic(function()
+		if stopped then
+			return
+		end
+		if (not started) then
+			-- options are not available in startplugin, so we load the history here
+			local historypath = emu.subst_env(manager.ui.options.entries.historypath:value():match("([^;]+)"))
```

Comment:
That’s not what `historypath` is for – you’re supposed to use the somewhat poorly-named “home path” for Lua plugin data.